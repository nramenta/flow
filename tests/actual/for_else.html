{% for i in [] %}
not gonna happen
{% else %}
else happened because we tried to iterate over an empty array
{% endfor %}

{% for i in null %}
null is iterable
{% else %}
can't iterate over nulls
{% endfor %}

{% for i in 0 %}
zero is iterable
{% else %}
can't iterate over numbers
{% endfor %}

{% for i in "foobar" -%}
{{- i }}
{% else %}
strings are not iterable
{% endfor %}
