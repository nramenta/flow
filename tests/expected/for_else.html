
else happened because we tried to iterate over an empty array



can't iterate over nulls



can't iterate over numbers



strings are not iterable

